### ISSUE TITLE
Workaround for 16-Colors with Truecolor

### ISSUE THREAD

**Issue by Gammer0909** [tomakita/Colorful.Console#93]
(If this has been brought up before and rejected, I apologize, but I could not find a similar issue)

Hi, I'm new to this project, and I noticed at the bottom of the README that it says
> Colorful.Console can only write to the console in 16 different colors
And I considered a workaround!

Basically, you can avoid using `System.ConsoleColor`, because not only do most macOS and linux terminals support this, but [Windows does too!](https://devblogs.microsoft.com/commandline/24-bit-color-in-the-windows-console/#:~:text=we've%20updated%20the%20Windows,color%20is%20becoming%20more%20established.)

And that thing is:

### [Truecolor/256 Color support](https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797#color-codes)

Truecolor is 24-bit colors, meaning you can convert from a `System.Drawing.Color` to a custom color.

Basically, a Truecolor query is just prepending an [ANSI Color Code](https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797) to the string you want.

For example, in your terminal, if you write 
```bash
echo "\x1b[38;2;255;100;0mTRUECOLOR\x1b[0m\n"
```
It outputs:

![Screen Shot 2024-01-05 at 7 40 36 PM](https://github.com/tomakita/Colorful.Console/assets/103458991/884a3c4c-b6ef-4b2b-8646-c8b6453a97f9)

I think this is a versatile solution, as I've [taken a swing at it](https://github.com/Gammer0909/Console), given that you can use string interpolation.

Hope this helps, Gammer0909
